Biome 2.4.14 broken on Windows on ARM

### Environment information

<details>

```bash

CLI:
  Version:                      2.4.14
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           v25.8.2
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Error:                        The configuration schema version does not match the CLI version 2.4.14
  Status:                       Loaded with errors
  Path:                         C:\Users\Jazza\Documents\tafe-s1-2026\biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true
  HTML full support enabled:    true

Formatter:
  Format with errors:           false
  Indent style:                 Tab
  Indent width:                 2
  Line ending:                  LF
  Line width:                   100
  Attribute position:           Auto
  Bracket spacing:              true
  Includes:                     unset

JavaScript Formatter:
  Enabled:                      unset
  JSX quote style:              Double Quotes
  Quote properties:             As needed
  Trailing commas:              All
  Semicolons:                   Always
  Arrow parentheses:            Always
  Bracket spacing:              true
  Bracket same line:            false
  Quote style:                  Double Quotes
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           Auto

JSON Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Trailing Commas:              unset
  Expand lists:                 unset

CSS Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Quote style:                  unset

GraphQL Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Bracket spacing:              unset
  Quote style:                  unset

Workspace:
  Open Documents:               0

```

</details>

```bash

$ pnpm format

> biome-repro@0.0.0 format C:\Users\Jazza\Downloads\biome-repro-1778030377380
> biome format .

 ELIFECYCLE  Command failed with exit code 2816.
```

I haven't bisected since I don't have the time right now, but using 2.4.9 works.

<details>

```bash

Jazza@Jazza-X-Elite CLANGARM64 ~/Downloads/biome-repro-1778030377380 (master)
$ pnpm ls
Legend: production dependency, optional only, dev only

biome-repro@0.0.0 C:\Users\Jazza\Downloads\biome-repro-1778030377380 (PRIVATE)
│
│   devDependencies:
├── @biomejs/biome@2.4.9
└── @biomejs/cli-win32-arm64@2.4.9

2 packages

Jazza@Jazza-X-Elite CLANGARM64 ~/Downloads/biome-repro-1778030377380 (master)
$ pnpm format

> biome-repro@0.0.0 format C:\Users\Jazza\Downloads\biome-repro-1778030377380
> biome format .

package.json format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Formatter would have printed the following content:

    14 14 │             "@biomejs/cli-win32-arm64": "2.4.9"
    15 15 │     }
    16    │ - }
       16 │ + }
       17 │ +


Checked 3 files in 2ms. No fixes applied.
Found 1 error.
format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  × Some errors were emitted while running checks.


 ELIFECYCLE  Command failed with exit code 1.

```

</details>

### Configuration

```JSON
{
	"$schema": "https://biomejs.dev/schemas/2.4.14/schema.json",
	"vcs": { "enabled": true, "clientKind": "git", "useIgnoreFile": true },
	"files": { "ignoreUnknown": false },
	"formatter": {
		"enabled": true,
		"formatWithErrors": false,
		"indentStyle": "tab",
		"indentWidth": 2,
		"lineEnding": "lf",
		"lineWidth": 100,
		"attributePosition": "auto",
		"bracketSameLine": false,
		"bracketSpacing": true,
		"expand": "auto",
		"useEditorconfig": true
	},
	"linter": { "enabled": true, "rules": { "recommended": true } },
	"javascript": {
		"formatter": {
			"jsxQuoteStyle": "double",
			"quoteProperties": "asNeeded",
			"trailingCommas": "all",
			"semicolons": "always",
			"arrowParentheses": "always",
			"bracketSameLine": false,
			"quoteStyle": "double",
			"attributePosition": "auto",
			"bracketSpacing": true
		}
	},
	"html": {
		"formatter": {
			"indentScriptAndStyle": false,
			"selfCloseVoidElements": "never",
			"enabled": true
		},
		"experimentalFullSupportEnabled": true
	},
	"assist": {
		"enabled": true,
		"actions": { "source": { "organizeImports": "on" } }
	}
}
```

### Playground link

https://github.com/jazzathedev/biome-repro-1778030377380

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

```json
	"devDependencies": {
		"@biomejs/biome": "https://pkg.pr.new/biomejs/biome/@biomejs/biome@df89c61",
		"@biomejs/cli-win32-arm64": "https://pkg.pr.new/biomejs/biome/@biomejs/cli-win32-arm64@df89c61"
	}
```

<details>

```bash
PS C:\Users\Jazza\Downloads\biome-repro-1778030377380> pnpm ls
Legend: production dependency, optional only, dev only

biome-repro@0.0.0 C:\Users\Jazza\Downloads\biome-repro-1778030377380 (PRIVATE)
│
│   devDependencies:
├── @biomejs/biome@2.4.14
└── @biomejs/cli-win32-arm64@2.4.14

2 packages
PS C:\Users\Jazza\Downloads\biome-repro-1778030377380> pnpm format

> biome-repro@0.0.0 format C:\Users\Jazza\Downloads\biome-repro-1778030377380
> biome format .

/c/Users/Jazza/Downloads/biome-repro-1778030377380/node_modules/.bin/biome: line 2: sed: command not found
/c/Users/Jazza/Downloads/biome-repro-1778030377380/node_modules/.bin/biome: line 2: dirname: command not found
/c/Users/Jazza/Downloads/biome-repro-1778030377380/node_modules/.bin/biome: line 4: uname: command not found
node:internal/modules/cjs/loader:1478
  throw err;
  ^

Error: Cannot find module 'C:\@biomejs\biome\bin\biome'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1475:15)
    at wrapResolveFilename (node:internal/modules/cjs/loader:1048:27)
    at defaultResolveImplForCJSLoading (node:internal/modules/cjs/loader:1072:10)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1093:12)
    at Module._load (node:internal/modules/cjs/loader:1261:25)
    at wrapModuleLoad (node:internal/modules/cjs/loader:255:19)
    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
    at node:internal/main/run_main_module:33:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v25.8.2
 ELIFECYCLE  Command failed with exit code 1.
PS C:\Users\Jazza\Downloads\biome-repro-1778030377380> & "C:\Users\Jazza\Downloads\biome-repro-1778030377380\node_modules\.pnpm\@biomejs+cli-win32-arm64@ht_916635578c95beae50413c98538f3781\node_modules\@biomejs\cli-win32-arm64\biome.exe" version
CLI:
Server:     not connected
PS C:\Users\Jazza\Downloads\biome-repro-1778030377380> & "C:\Users\Jazza\Downloads\biome-repro-1778030377380\node_modules\.pnpm\@biomejs+cli-win32-arm64@ht_916635578c95beae50413c98538f3781\node_modules\@biomejs\cli-win32-arm64\biome.exe" format --write
PS C:\Users\Jazza\Downloads\biome-repro-1778030377380>
```

</details>

It was very hard to find the right exe but I'm fairly sure its right, the version command as u can see obviously didnt work which i think means its the pre release version

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -126,7 +126,7 @@ indexmap                     = { version = "2.14.0" }
 insta                        = "1.47.2"
 jiff                         = "0.2.24"
 libc                         = "0.2.186"
-mimalloc                     = "0.1.50"
+mimalloc                     = "0.1.48"
 papaya                       = "0.2.4"
 path-absolutize              = { version = "3.1.1", features = ["use_unix_paths_on_wasm"], optional = false }
 phf                          = { version = "0.13.1", features = ["macros"] }

